src/client/components/nutrients-table/nutrients-table.vue

```diff
@@ -146,14 +166,36 @@ export default {
   .nutrients-table__skeleton-value {
     width: 2.5em;
   }
+
+  .nutrients-table__trend {
+    display: inline-block;
+    width: 55px;
+    font-size: .8em;
+  }
+  .nutrients-table__trend--down::after {
+    display: inline-block;
+  }
+  .nutrients-table__trend--down {
+    color: #0f9d58;
+  }
+  .nutrients-table__trend--down::after {
+    content: "▼";
+  }
+  .nutrients-table__trend--up {
+    color: #d23f31;
+  }
+  .nutrients-table__trend--up::after {
+    content: "▲";
+  }
+
```

Comment:
We need te adjust the media query for data-table__content.nutrients-table__content--nutrient to 157px or just remove the responsiveness altogether and stick with a 512px width which is ideal for ipad and most safe.

In case we remove we can also simplify nutrients-totals.vue